bosh-release/README.md

```diff
@@ -46,7 +57,7 @@ A BOSH release for deploying WSO2 API Manager 2.1.0 on BOSH Director:
 
 6. Download Oracle JDK 1.8 from Oracle website and WSO2 API Manager 2.1.0 via WSO2 Update Manager (WUM).
 
-7. Add above distributions as blobs:
+7. Add above distributions as blobs. Go inside `wso2-apim-bosh-release` and run below commands.
```

Comment:
Would it be better to mention this as a command? Then the user can directly copy and execute it. WDYT?

Reply:
+1, I'll add.